sand/controllers/transform.py

```diff
@@ -0,0 +1,190 @@
+import json
+import re
+import sys
+import traceback
+from typing import Dict, List, Union
+
+from flask import jsonify, request
+from flask.blueprints import Blueprint
+from RestrictedPython import compile_restricted_function, safe_globals
+from werkzeug.exceptions import BadRequest
+
+from sand.models.table import Link, Table, TableRow
+from sand.models.transform import Context
+
+transform_bp = Blueprint("transform", "transform")
+USER_DEFINE_FUNCTION_NAME = ""
+
+
+def transform_map(transform_func, data, tolerance):
+    transformed_data = []
+    exec_tolerance = tolerance
+    for path, value, context in data:
+        value = value[0]
+        tdata = dict()
+        tdata["path"] = path
+        tdata["value"] = value
+        try:
+            result = transform_func(value, context)
+            tdata["ok"] = result
+            transformed_data.append(tdata)
```

Comment:
duplicated with line 35. resolve this by changing return statement inside the loop to break